n8n-io/n8n
# feat(Embeddings Cohere Node): Add v3 Cohere models

## Summary

Just a simple PR updating the `EmbeddingsCohere` node to include the latest models for generating text embeddings. It adds new model options (Cohere Embedding V3 English + Multilang...).

**Changes made:**
- Added new model options: `embed-english-v3.0`, `embed-english-light-v3.0`, `embed-multilingual-v3.0`, `embed-multilingual-light-v3.0`.
- Updated the `supplyData` method to use the new models and API format.

**How to test:**
1. Configure the `EmbeddingsCohere` node with appropriate credentials.
2. Select one of the new models from the dropdown menu.
3. Execute the node and verify that embeddings are generated correctly.

## Related Linear tickets, Github issues, and Community forum posts

- None that I am aware of, just noticed it on the node properties and quickly edited the code to change the model.

## Review / Merge checklist
